sktime/base/_meta.py

```diff
@@ -17,14 +17,6 @@ class _HeterogenousMetaEstimator(BaseEstimator):
     Partly adapted from sklearn utils.metaestimator.py.
     """
 
-    def get_params(self, deep=True):
-        """Return estimator parameters."""
-        raise NotImplementedError("abstract method")
-
-    def set_params(self, **params):
```

Comment:
If this is causing issues with get/set params in children - maybe worth making in another PR to make sure it gets merged?

Reply:
precisely what I also thought when I worked on this PR, and that's why I created this other PR https://github.com/sktime/sktime/pull/3708

Glad to see we think along the same lines :-)

Reply:
👍